src/i18n.js

```diff
@@ -16,6 +17,7 @@ i18n
   .use(Backend)
   .use(LanguageDetector)
   .init({
+    load: 'currentOnly', // see https://github.com/i18next/i18next-http-backend/issues/61
```

Comment:
this is *supposed* to prevent the selection of multiple languages, but didn't, because i18n-http-backend doesn't know whether the code is valid until it goes to loadPath. The changes to `loadPath` below are where the real fix is.